src/routes/api/client-key/client-key-regex.test.ts

```diff
@@ -1,14 +1,32 @@
 import { extractClientKey } from "./client-key-regex";
 
 const sampleUuid = "yyyyyyyy-0569-4f33-8413-cccccccc";
+const sampleJiraSite = "sample-site.atlassian.net";
+const xmlTemplate = `
+<consumer>
+<key>$$KEY$$</key>
+<name>JIRA</name>
+<publicKey>
+...SIb3DQEBAQUAA4GNADCBiQK...
+</publicKey>
+<description>Atlassian JIRA at url</description>
+</consumer>
+`;
+
+const getXml = (key: string) => {
+	return xmlTemplate.replace("$$KEY$$", key);
+};
```

Comment:
why are you adding a string replace when typescript already has templating built-in?

Reply:
Fixed.